dask/Dask_Hello_World.ipynb

```diff
@@ -0,0 +1,332 @@
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "This notebook was tested using the `nvcr.io/nvidia/rapidsai/rapidsai:0.5-cuda10.0-runtime-ubuntu18.04-gcc7-py3.7` Docker container from [NVIDIA GPU Cloud](https://ngc.nvidia.com) and run on the NVIDIA Tesla V100 GPU. Please be aware that your system may be different and you may need to modify the code or install packages to run the below examples. \n",
+    "\n",
+    "If you think you have found a bug or an error, please file an issue here: https://github.com/rapidsai/notebooks/issues\n",
+    "\n",
+    "Before we begin, let's check out our hardware setup by running the `nvidia-smi` command."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!nvidia-smi"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, let's see what CUDA version we have:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!nvcc --version"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Lbraries\n",
+    "\n",
+    "Next, let's load some libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dask; print('Dask Version:', dask.__version__)\n",
+    "from dask.delayed import delayed\n",
+    "from dask.distributed import Client\n",
+    "import os\n",
+    "import subprocess\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup Dask\n",
+    "\n",
+    "Dask is a library the allows for parallelized computing. Written in Python, it allows one to schedule tasks and do so dynamically as well handle large data structures - similar to those found in NumPy and Pandas. In the subsequent tutorials, we'll show how to use Dask with Pandas and cuDF and how we can use both to accelerate common ETL tasks as well as build ML models like XGBoost.\n",
+    "\n",
+    "To learn more about Dask, check out the documentation here: http://docs.dask.org/en/latest/\n",
+    "\n",
+    "Dask operates using a concept of a \"Client\" and \"workers\". The client tells the workers what tasks to perform and when to perform. Typically, we set the number of works to be equal to the number of computing resources we have available to us. For example, wer might set `n_workers = 8` if we have 8 CPU cores on our machine that can each operate in parallel. This allows us to take advantage of all of our computing resources and enjoy the most benefits from parallelization.\n",
+    "\n",
+    "Dask is a first class citizen in the world of General Purpose GPU computing and the RAPIDS ecosystem makes it very easy to use Dask with cuDF and XGBoost. As we see below, we can inititate a Cluster and Client using only few lines of code."
```

Comment:
Can you add something here pointing to [dask-cuda](https://github.com/rapidsai/dask-cuda) and [dask-cudf](https://github.com/rapidsai/dask-cudf)?

Mention that they're both being updated frequently, and to please open issues there if seeing unexpected behavior.

Reply:
Sorry, I misunderstood the intent of this notebook was to introduce _just_ Dask, not dask + RAPIDS